.vscode/settings.json

```diff
@@ -4,6 +4,8 @@
   "[typescript]": {
       "editor.formatOnSave": true
   },
+  "prettier.trailingComma": "es5",
+  "prettier.arrowParens": "always",
```

Comment:
Please move this in a separate prettier configuration files.
Then we can copy that file to all other projects.

Reply:
The file was created. This file should also be added to our repository templates, so everybody has the same prettier configuration.